tutorials/plot_visualize_evoked.py

```diff
@@ -23,20 +23,21 @@
 ###############################################################################
 # Notice that ``evoked`` is a list of :class:`evoked <mne.Evoked>` instances.
 # You can read only one of the categories by passing the argument ``condition``
-# to :func:`mne.read_evokeds`. To make things more simple for this tutorial, we
-# read each instance to a variable.
-evoked_l_aud = evoked[0]
-evoked_r_aud = evoked[1]
-evoked_l_vis = evoked[2]
-evoked_r_vis = evoked[3]
+# to :func:`mne.read_evokeds`.
+# The read evoked objects can be stored in a dict for easier access.
+conditions = ["Left Auditory", "Right Auditory", "Left visual", "Right visual"]
+evoked = dict()
+for condition in conditions:
+    evoked[condition] = mne.read_evokeds(fname, baseline=(None, 0),
+                                          proj=True, condition=condition)
```

Comment:
I didn't take @agramfort and @dengemann comments as a no to the whole PR (but I may be wrong :) ) - I is useful to teach newcomers to store their stuff in dicts with meaningful keys. I would just change the `for ii, (condition, axis) in enumerate(zip(conditions, ax))` loop to be simpler (especially that `ii` is not used within the loop). Alternatively - you could also revert changes made in this PR and just add a note about usefulness of storing data in dictionaries.